labelCloud/io/segmentations/numpy.py

```diff
@@ -0,0 +1,21 @@
+from .base import BaseSegmentationHandler
+from pathlib import Path
+import numpy as np
+import numpy.typing as npt
+
+
+class NumpySegmentationHandler(BaseSegmentationHandler):
+    EXTENSIONS = {".bin"}
+
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+
+    def _create_labels(self, num_points: int) -> npt.NDArray[np.int8]:
+        return np.ones(shape=(num_points,), dtype=np.int8) * self.default_label
```

Comment:
Is it standard for points without label to set 0 or could it also be None/ NA/ ...?

Reply:
Not sure if it's a standard but it's common to have a "clutter" class to indicate points that are not of interest and I think in practice it's common to pre-label everything as "clutter" and gradually label points to other classes. In my opinion, it's also easier to treat all the labels as integer so it's easier to manipulate them (i.e. we don't need a if na ... else ... logic)